spec/features/admin/shipping_methods_spec.rb

```diff
@@ -23,12 +23,14 @@
       page.should have_field 'shipping_method_name'
       page.should have_field 'shipping_method_description'
       page.should have_select 'shipping_method_display_on'
-      page.should have_field 'shipping_method_require_ship_address_true', checked: true
```

Comment:
you should still verify the require_ship_address is there, right?

Reply:
"the choice between Delivery and Pick Up, which are shipping categories now"
that's a very interesting assumption! because it would be good to get rid of this field. 
but I dont think it's correct. afaik in ofn v2 we will keep require_ship_address.

Reply:
I believe we will keep it, yes